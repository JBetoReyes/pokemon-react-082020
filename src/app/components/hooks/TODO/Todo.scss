.complete {
  text-decoration: line-through;
}

.list-group-item {
  display: flex;
  align-items: center;
}
