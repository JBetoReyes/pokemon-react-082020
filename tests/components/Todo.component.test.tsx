import React from 'react';
import Todo from '@components/hooks/TODO/TodoList.component';
import { shallow } from 'enzyme';

describe('Todo', () => {
  let todoWrapper = shallow(<Todo />);
  beforeEach(() => {
    todoWrapper = shallow(<Todo />);
  });
  test('Should match with the snapshot', () => {
    expect(todoWrapper).toMatchSnapshot();
  });
});
