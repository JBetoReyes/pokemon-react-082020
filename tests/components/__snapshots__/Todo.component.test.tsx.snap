// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo Should match with the snapshot 1`] = `
<Fragment>
  <h1>
    TODO List
  </h1>
  <hr />
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-6  todo-list"
      >
        <ol
          className="list-group list-group-flush"
        />
      </div>
      <div
        className="col-5 todo-form"
      >
        <h4>
          Add TODO
        </h4>
        <form
          onSubmit={[Function]}
        >
          <input
            className="form-control"
            name="newTodo"
            onChange={[Function]}
            type="text"
            value=""
          />
          <button
            className="mt-1 btn btn-outline-primary btn-block"
            type="submit"
          >
            Add
          </button>
        </form>
      </div>
    </div>
  </div>
</Fragment>
`;
